Ugizwenayo-Divine/FoodStuffPriceCompare-App
# setting the travis

#### What does this PR do?
- setting up travis
#### Description of Task to be completed?
- added the repo on travis
- created the .travis.yml file
- configured the the .travis.yml file
#### How should this be manually tested?
- After cloning the project repo, run npm i inside the cloned repo folder
- change in cloned repo
- git fetch
- git checkout ch-setup-babel
- npm i
- must see the build badge
#### Any background context you want to provide?
- you must have node installed locally
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
N/A
